### PR TITLE
Added in copying text from clipboard.

### PR DESCRIPTION
**Pull request by lwinston7**
@gtkesh

### CODE REVIEW

File: app/src/main/java/com/example/gtkesh/smartsignservice/SmartSignAccessibilityService.java [hunk 1]
```diff
@@ -93,6 +99,11 @@ protected boolean onGesture(int gestureId) {
 
         if(gestureId == AccessibilityService.GESTURE_SWIPE_UP_AND_LEFT){
             Log.d("YAAAAAY", "GESTURE DETECTED!");
+
+            ClipboardManager clipboardManager = (ClipboardManager)getSystemService(CLIPBOARD_SERVICE);
```
Comment (gtke) [inline app/src/main/java/com/example/gtkesh/smartsignservice/SmartSignAccessibilityService.java]: Do we need `ClipboardManager` here? We're not getting the text from the clipboard on `onGesture`. When `onGesture` is called it is assumed that there should be a text grabbed already. So maybe we dont need this here and have a global variable and set it once we grab it wherever that happens.

Comment (lwinston7) [inline app/src/main/java/com/example/gtkesh/smartsignservice/SmartSignAccessibilityService.java]: Yes, it's the only way to get copied text from the clipboard in Android.

Comment (gtke) [inline app/src/main/java/com/example/gtkesh/smartsignservice/SmartSignAccessibilityService.java]: @lwinston7 Agreed. But we already have it in `onAccessibilityEvent` which does exactly what we need.

**Comment by lwinston7** [comment]
You're right, this should be cleaner now.

**Comment by gtke** [comment]
:+1:

**Comment by gtke** [comment]
Looks good. Merging.

**Pull request merged**